skipper.env

```diff
@@ -79,7 +79,9 @@ OLM_OPERATORS
 KUBE_API
 INSTALLER_KUBECONFIG
 DEBUG
+FOCUS
```

Comment:
what does this variable means?

Reply:
It is the variable that allows focusing on a single ginko test. It has nothing to do with HYPERTHRADING but since I am changing the file already it is an important addition to ease developing cycle